### ISSUE TITLE
Autocomplete is not read out issue 

### ISSUE THREAD

**Issue by RagupathyR** [nvaccess/nvda#16140]
Summary:
The NVDA does not mention the autocomplete attribute, although the fact that the input field contains autocomplete with the proper token.

Sample Code:
![image](https://github.com/nvaccess/nvda/assets/24953023/ab5af93c-155a-41f5-9955-10a7f5c302e2)

Steps to reproduce:
Navigate to the Billing address input field when the NVDA screen reader is on.

Actual behavior:
On accessing the "Billing address" input field, the screen reader is announced as "Billing address edit blank".

Expected behavior:
The Autocomplete attribute and token is not announced by the screen reader.

NVDA version: 2023.3.3 (2023.3.3.30854)
Browser: Chrome (Version 121.0.6167.140 (Official Build) (64-bit))

Windows version:
Edition : Windows 11 Enterprise
Version : 22H2

Does the issue still occur after restarting your computer?
Yes.

**Comment by hwf1324** [comment]
HI, since most of the people here are visually impaired, please don't use images to describe the content, you can use some of the platforms that share code examples to share your use cases.

**Comment by Adriani90** [comment]
Closing as invalid. Please provide clear steps to reproduce, a minimal test case on codepen for example, and clear expected behavior with the exact speech sequence you expect to hear when using NVDA.
